cypress/tests/upstream/vm-actions-confirm.cy.ts

```diff
@@ -2,15 +2,8 @@ import { Example } from '../../utils/const/string';
 import { tab } from '../../views/tab';
 
 describe('Test VM actions confirm', () => {
-  before(() => {
-    cy.login();
-    cy.visit('');
-    cy.switchToVirt();
-  });
-
-  it('navigate to settings', () => {
-    cy.visitCatalogVirt();
-    cy.visitOverviewVirt();
+  before('navigate to settings', () => {
```

Comment:
Is it convention to start 'before' descriptions with a lowercase letter or is this a typo?

Reply:
I don't know if there is a convention.
I keep the uppercase in description and lowercase in others.